README.md

```diff
@@ -93,57 +93,126 @@ dicom-anonymizer InputFilePath OutputFilePath --dictionary dictionary.json
-	print('custom action 1')
-def ActionNameNotReferenced2(dataset, tag):
-	print('custom action 2')
-
-actionsTagMap = {
-	(0x0001, 0x0001): "keep", #will use the default method in the dicom anonymizer
-	(0x0002, 0x0002): ActionNameNotReferenced,
-	(0x0003, 0x0003): ActionNameNotReferenced2
-}
-
-# Generate a custom dictionary that will be override the default one
-actionsMap = dicomanonymizer.anonymizer.generateActionsDictionary(actionsTagMap)
-
-# Send the map to the main function
-dicomanonymizer.anonymizer.anonymize(actionsMap)
+import argparse
+from dicomanonymizer import *
+
+def main():
+    parser = argparse.ArgumentParser(add_help=True)
+    parser.add_argument('input', help='Path to the input dicom file or input directory which contains dicom files')
+    parser.add_argument('output', help='Path to the output dicom file or output directory which will contains dicom files')
+    parser.add_argument('--suffix', action='store', help='Suffix that will be added at the end of series description')
+    args = parser.parse_args()
+
+    InputPath = args.input
+    OutputPath = args.output
+
+    dictionary = {}
+
+    def setupSeriesDescription(dataset, tag):
+        element = dataset.get(tag)
+        if element is not None:
+            element.value = element.value + '-' + args.suffix
+
+    # Store all tags into one array and setup the action in
+    # order to keep these value as it is
+    # All tags are defined on file dicomfields.py
+    allTags = []
+    allTags.extend(D_TAGS)
+    allTags.extend(Z_TAGS)
+    allTags.extend(X_TAGS)
+    allTags.extend(U_TAGS)
+    allTags.extend(Z_D_TAGS)
+    allTags.extend(X_Z_TAGS)
+    allTags.extend(X_D_TAGS)
+    allTags.extend(X_Z_D_TAGS)
+    allTags.extend(X_Z_U_STAR_TAGS)
+
+    # the 'keep' method is already defined into the dicom-anonymizer
+    # It will overrides the default behaviour
+    for i in allTags:
+        dictionary[i] = keep
+
+    if args.suffix:
+        dictionary[(0x0008, 0x103E)] = setupSeriesDescription
+
+    # Launch the anonymization
+    anonymize(InputPath, OutputPath, dictionary)
+
+if __name__ == "__main__":
+    main()
 ```
 
 In your own file, you'll have to define:
 - Your custom functions. Be careful, your functions always have in inputs a dataset and a tag
-- A dictionary which map your functions to a string
+- A dictionary which map your functions to a tag
+
+## Anonymize dicom tags without dicom file
 
-# Overrides actions for default tags
-You can have the need to keep some tags, for example UID. Then, instead of replace it (`replaceUID` action) then you can override it :
+For some reason, you need to anonymize dicom fields without initial dicom file (extracted from a database for example). Here is how you can do it:
+```python
+from dicomanonymizer import *
+
+def main():
+
+  # Create a list of tags object that should contains id, type and value
+  fieldsList = [
+    { # Replaced by Anonymized
+      'id': '(0x0040, 0xA123)',
```

Comment:
why bother store it as a string ? why not simply a tuple ?

Reply:
Because it's a string in the project. Not a tuple.

Reply:
ok, this special case should not be in this example.

Reply:
Why ? As I already encountered the issue, why I cannot share it in the doc ?

Reply:
because it is an implementation detail. That brings a bit of noise when reading this high level documentation. 

Now if you feel strongly about it, you can leave it.